src/main/kotlin/net/atos/zac/smartdocuments/templates/model/SmartDocumentsTemplate.kt

```diff
@@ -15,17 +15,17 @@ import nl.lifely.zac.util.AllOpen
 import java.time.ZonedDateTime
 
 @Entity
-@Table(schema = FlywayIntegrator.SCHEMA, name = "template")
+@Table(schema = FlywayIntegrator.SCHEMA, name = "document_creation_template")
```

Comment:
Personally I would still prefer e.g. `smartdocuments_document_creation_template` because we are not building a generic document creation solution and I honestly do not know if we ever will. Only if we do so we should try to make things generic in my opinion. Still, if there is really nothing SD specific about this datamodel (but how do we even know this?) I think it's fine.

Reply:
Renamed to `smartdocuments_document_creation_template` and `smartdocuments_document_creation_template_group`